### PR TITLE
[PE-4751] Move global-nav styles into inverse sidenav styles

### PR DESCRIPTION
**Pull request by jllr**
https://ctl.atlassian.net/browse/PE-4751

### CODE REVIEW

**Comment by allyourbotsarebelongtous** [comment]
Your CHI Pull Request website is available for review at <a href="https://4b74715e9ea32894dca30db7e7d467b8efd1a233-chi-pr.enterprise.ctl.io" target="_blank"> https://4b74715e9ea32894dca30db7e7d467b8efd1a233-chi-pr.enterprise.ctl.io </a>

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/903/ for more details.

**Comment by mattnickles** [comment]
@jllr Wow this is looking great! Found one minor issue in the base styles for the -active -unselected drawer list item.
image

File: src/chi/components/sidebar/sidenav.scss [hunk 1]
```diff
@@ -55,39 +54,161 @@ $sidenav-lg-width: 12.5rem;
...
+      a.-active {
+        background-color: set-color(grey, 10);
+        box-shadow: inset 0.125rem 0 0 set-color(grey, 100);
+      }
     }
 
     & > a {
       &.-active { // Drawer shown, will sets active the trigger (this anchor element)
-        background-color: set-color(grey, 20);
+        background-color: set-color(grey, 10);
```
Comment (mattnickles) [inline src/chi/components/sidebar/sidenav.scss:82]: This line can be removed to ensure consistent active behavior with inverse/global nav. In the inverse version, the active item bg stays green when its submenu drawer is triggered. This line is causing the base versions active item bg to change from blue to grey when its submenu drawer is triggered.

**Comment by allyourbotsarebelongtous** [comment]
Your CHI Pull Request website is available for review at <a href="https://0de0a53ac5f5d3ae849132286f4a3599e8b147db-chi-pr.enterprise.ctl.io" target="_blank"> https://0de0a53ac5f5d3ae849132286f4a3599e8b147db-chi-pr.enterprise.ctl.io </a>

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/905/ for more details.

File: src/chi/components/sidebar/sidenav.scss [hunk 2]
```diff
@@ -55,39 +54,161 @@ $sidenav-lg-width: 12.5rem;
...
+              .chi-sidenav__title {
+                .chi-icon {
+                  margin-left: 0.5rem;
+                }
+              }
+            }
+
+            &.-active {
+              & > a {
+                background-color: set-color(grey, 90);
```
Comment (mattnickles) [inline src/chi/components/sidebar/sidenav.scss:129]: This is the dark background-color causing the conflict in the base version. Change to `background: none;` and change the color to `color: $primary-color;`

**Comment by allyourbotsarebelongtous** [comment]
Your CHI Pull Request website is available for review at <a href="https://b513da8d9d455382760ce8d0374e0c503733adf4-chi-pr.enterprise.ctl.io" target="_blank"> https://b513da8d9d455382760ce8d0374e0c503733adf4-chi-pr.enterprise.ctl.io </a>

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/907/ for more details.

**Comment by allyourbotsarebelongtous** [comment]
The following instances are being terminated: https://ops.elasticbox.com/services/instances/i-yy6xbe, https://ops.elasticbox.com/services/instances/i-jz2gsc, https://ops.elasticbox.com/services/instances/i-d54jff

**Pull request merged**